sound/soc/codecs/rt722-sdca-sdw.c

```diff
@@ -201,6 +201,8 @@ static int rt722_sdca_read_prop(struct sdw_slave *slave)
 	unsigned long addr;
 	struct sdw_dpn_prop *dpn;
 
+	sdw_slave_read_prop(slave);
+
```

Comment:
that makes no sense, why would we read the information twice, here and in the previous patch?

Reply:
No, the previous commit add lane mapping reading to sdw_slave_read_prop(), this function also reads other DisCo properties. Currently, No codec SDW driver implementation calls  sdw_slave_read_prop() to read DisCo properties, I add the call for rt722-sdca, because it need to read lane mapping.

Reply:
No, the previous commit adds lane mapping reading to sdw_slave_read_prop(), this function also reads other DisCo properties. Currently, no codec SDW driver implementation calls  sdw_slave_read_prop() in its read_prop() ops to read DisCo properties, I add the call for rt722-sdca, because it needs to read lane mapping.

Maybe we should inform codec vender about this?

Reply:
Humm, that's fascinating that we wrote a function that no one uses. I thought the plan was to first use sdw_slave_read_prop() and then add vendor-specific parts, as we do for the manager.

````
static int intel_prop_read(struct sdw_bus *bus)
{
	/* Initialize with default handler to read all DisCo properties */
	sdw_master_read_prop(bus);

	/* read Intel-specific properties */
	sdw_master_read_intel_prop(bus);

	return 0;
}
````

oh well. I guess this will have to change for more SDCA support, but for now let's go with the path of least resistance.